pontoon/teams/static/js/bugzilla.js

```diff
@@ -1,102 +1,113 @@
-var Pontoon = (function (my) {
-  return $.extend(true, my, {
-    bugzilla: {
+const Pontoon = ((my => $.extend(true, my, {
+  bugzilla: {
 
-      /*
-       * Retrieve bugs for the given locale and update bug count and tab content
-       * using the provided elements and callbacks.
-       * 
-       * Heavily inspired by the similar functionality available in Elmo.
-       * 
-       * Source: https://github.com/mozilla/elmo/blob/master/apps/bugsy/static/bugsy/js/bugcount.js
-       * Authors: Pike, peterbe, adngdb
-       */
-      getLocaleBugs: function (locale, container, tab, countCallback, errorCallback) {
-        return $.ajax({
-          url: 'https://bugzilla.mozilla.org/rest/bug',
-          data: {
-            'field0-0-0': 'component',
-            'type0-0-0': 'regexp',
-            'value0-0-0': '^' + locale + ' / ',
-            'field0-0-1': 'cf_locale',
-            'type0-0-1': 'regexp',
-            'value0-0-1': '^' + locale + ' / ',
-            'resolution': '---',
-            'include_fields': 'id,summary,creation_time,last_change_time'
-          },
-          success: function(data) {
-            if (data.bugs.length) {
-              data.bugs.sort(function(l, r) {
-                return l.last_change_time < r.last_change_time ? 1 : -1;
-              });
+    /*
+     * Retrieve bugs for the given locale and update bug count and tab content
+     * using the provided elements and callbacks.
+     * 
+     * Heavily inspired by the similar functionality available in Elmo.
+     * 
+     * Source: https://github.com/mozilla/elmo/blob/master/apps/bugsy/static/bugsy/js/bugcount.js
+     * Authors: Pike, peterbe, adngdb
+     */
+    //modification by Okpo starts here
+    //current fields on pontoon are: ID, SUMMARY, LAST CHANGED, DATE CREATED
+    // add ASSIGNEE and STATUS to the current list of bugs....
```

Comment:
Please don't leave comments needed for review withing the code. Use GitHub comments if needed instead. Thanks!

Reply:
okay, thanks mathjazz. I will edit as suggested. Thanks